templates/vue/src/components/modals/common/FileUpload.vue

```diff
@@ -230,30 +230,24 @@ export default {
   },
   data() {
     return {
+      // NOTE: compactMode cannot work without showing the image preview
+      showImagePreviewValue: this.compactMode || this.showImagePreview,
```

Comment:
When is `showImagePreviewValue` used in this component?

Reply:
Oops, forgot to update its references. Fixed now.